config/secrets.yml

```diff
@@ -1,35 +0,0 @@
-# Be sure to restart your server when you modify this file.
-
-# Your secret key is used for verifying the integrity of signed cookies.
-# If you change this key, all old signed cookies will become invalid!
-
-# Make sure the secret is at least 30 characters and all random,
-# no regular words or you'll be exposed to dictionary attacks.
-# You can use `rails secret` to generate a secure secret key.
-
-# Make sure the secrets in this file are kept private
-# if you're sharing your code publicly.
-
-# Shared secrets are available across all environments.
-
-# shared:
-#   api_key: a1B2c3D4e5F6
-
-# Environmental secrets are only available for that specific environment.
-
-development:
-  encryption_key: someRandomEncryptionKeyForDev123
-  secret_key_base: some_random_dev_key
-
-test:
-  encryption_key: someRandomEncryptionKeyForTest12
-  secret_key_base: some_random_test_key
-
-# Do not keep production secrets in the unencrypted secrets file.
-# Instead, either read values from the environment.
-# Or, use `bin/rails secrets:setup` to configure encrypted secrets
-# and move the `production:` environment over there.
-
-production:
-  encryption_key: <%= ENV["ENCRYPTION_KEY"] %>
```

Comment:
The issue here is through deleting this file, and the associated config, it introduces breaking changes in `user.rb` and `git_hub/service.rb`.

I assume that the Rails "auto-updater" removed this file, and the config changes around secrets in the config directory, correct?

We'll need to find a working solution with the new credentials management system in 5.2, or some other solution we're happy with, before merging this.